packages/11ty/_plugins/transforms/outputs/epub/manifest.js

```diff
@@ -118,6 +118,13 @@ module.exports = (eleventyConfig) => {
     ? removeHTML(full).replace(/\r?\n|\r/g, ' ')
     : oneLine
 
+  /**
+   * Strip milliseconds from ISO date string (`.sss`)
+   */
+  const pubDateWithoutMs = pubDate
+    .toISOString()
+    .replace(/\.\d{3}/, '')
```

Comment:
I know this could be done more expressively with something like `moment.js` but I wasn't sure if we wanted to add a dependency for just one thing